server/container/guice/cassandra-guice/src/test/java/org/apache/james/GuiceModuleTestExtension.java

```diff
@@ -0,0 +1,33 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james;
+
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+import com.google.inject.Module;
+
+public interface GuiceModuleTestExtension {
+
+    Module getModule();
+
+    default void registerSelfToContext(ExtensionContext context) {
```

Comment:
anything related to guice here ?

Reply:
nope, actually I don't need this class, but i want to create an extension mirror version of existing `GuiceModuleTestRule`. Then an extension mirror version of `DockerCassandraRule` and `DockerElasticSearchRule`, to easy replace those `DockerCassandraRule` & `DockerElasticSearchRule` by new Junit 5 extensions in the future.